src/page/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
boolean const should start with 'is' or 'has'

```suggestion
const [loading, setLoading] = useState(false);
const [isLoading, setIsLoading] = useState(false);

Reply:
> boolean const should start with 'is' or 'has'

Good point